services/webhooks/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
As far as i know this service should  in OIH is called `communication-router`. Am i right?